tools/custom-roles-analyzer/README.md

```diff
@@ -67,18 +72,17 @@ This will create an executable jar inside the `target` folder which will be used
 ## Running the tool
 Run following command:
 ```
-analyze.sh --org <org_id> --format <format> --role-analysis --binding-analysis
+analyze.sh --org <org_id> --format <format> --role-analysis
 ```
 Parameters: \
 --org       : GCP Organization Id (mandatory parameter). \
 --format    : Result format (optional parameter; defaults to csv and supports json). \
 --role-analysis      : Run custom role analysis (Optional parameter; default to custom role analysis). \
---binding-analysis   : Run role binding analysis (Optional parameter; default to custom role analysis).
 You can simply run as follows to get default format and custom role analysis:
 ```
 analyze.sh --org <org_id>
 ```
- 
+<!-- 
 You can run tool as follows to find the role bindings:
```

Comment:
Why is this commented out? Is it not valid anymore? I think those options still work.

Reply:
Those options will work until I move the code to different tool.